src/MetalPerformanceShaders/MPSCompat.cs

```diff
@@ -38,13 +38,28 @@ public virtual MPSImage EncodeToCommandBuffer (IMTLCommandBuffer commandBuffer,
 	}
 
 	public partial class MPSCnnConvolution {
-		[TV (11, 0), iOS (11, 0)]
+#if NET
```

Comment:
src/MetalPerformanceShaders/MPSCompat.cs is under !NET and a compat file, please revert.

Reply:
Great catch, thanks!